pyproject.toml

```diff
@@ -40,10 +40,11 @@ rouge-score = "^0.1.2"
 detoxify = "^0.5.2"
 langchain = "^0.3.7"
 asyncio = "^3.4.3"
-sentence-transformers = "^2.7.0"
+sentence-transformers = ">=3.10"
```

Comment:
This is motivated by the Snyk SCA scan, see https://security.snyk.io/vuln/SNYK-PYTHON-SENTENCETRANSFORMERS-8161344